_posts/2020-10-12-encyclopedia-influence-gospel-human-culture.md

```diff
@@ -7,7 +7,7 @@ author: JohnRDOrazio
 comments: true
 ---
 
-== Open knowledge ==
+# Open knowledge
```

Comment:
```suggestion
## Open knowledge

```

Better to use h2 instead of h1 inline in a post, and also make sure to add a line between headings and paragraphs.